pom.xml

```
@@ -6,7 +6,7 @@
 
   <groupId>de.digitalcollections</groupId>
   <artifactId>solr-ocrhighlighting</artifactId>
-  <version>0.9.0</version>
+  <version>0.9.0-SNAPSHOT</version>
```

Should this be part of the next release?
```suggestion
  <version>0.9.1-SNAPSHOT</version>
```